test/integration/with-polling.spec.ts

```diff
@@ -0,0 +1,451 @@
+import { TestArguments } from '@well-known-components/test-helpers'
+import { ethers } from 'ethers'
+import { Socket, io } from 'socket.io-client'
+import { AuthChain, Authenticator, AuthLinkType } from '@dcl/crypto'
+import { METHOD_DCL_PERSONAL_SIGN } from '../../src/ports/server/constants'
+import { MessageType } from '../../src/ports/server/types'
+import { BaseComponents } from '../../src/types'
+import { test, testWithOverrides } from '../components'
+
+let desktopClientSocket: HttpPollingClient
+let authDappSocket: Socket
+
+type HttpPollingClient = {
+  request(data: unknown): Promise<any>
+  poll(requestId: string): Promise<any>
+  cancel(): void
+}
+
+function createHttpPollingClient(url: string): HttpPollingClient {
+  let shouldPoll = true
+
+  return {
+    async request(data: unknown) {
+      // Make a post request
+      const response = await fetch(`${url}/http-polling/request`, {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: [['Content-Type', 'application/json']]
+      })
+
+      return response.json()
+    },
+    async poll(requestId: string) {
+      while (shouldPoll) {
+        const response = await fetch(`${url}/http-polling/${requestId}`)
+        if (response.status === 204) {
+          await new Promise(resolve => setTimeout(resolve, 1000))
+          continue
+        }
+
+        return response.json()
+      }
+    },
+    cancel() {
+      shouldPoll = false
+    }
+  }
+}
+
+afterEach(() => {
+  desktopClientSocket.cancel()
+  authDappSocket.close()
+})
+
+async function connectClients(args: TestArguments<BaseComponents>) {
+  const port = await args.components.config.getString('HTTP_SERVER_PORT')
+
+  desktopClientSocket = createHttpPollingClient(`http://localhost:${port}`)
+  authDappSocket = io(`http://localhost:${port}`)
+
+  await new Promise(resolve => {
+    const handleOnConnect = () => {
+      resolve(undefined)
+    }
+
+    authDappSocket.on('connect', handleOnConnect)
+  })
+}
+
+test('when sending a request message with an invalid schema (http-polling-version)', args => {
```

Comment:
Writing `(http-polling-version)` is redundant as the test runner will output the name of the file to identity the tests.

```
 PASS  test/integration/with-polling.spec.ts (37.869 s)
```